cmake/BCCompiler.cmake

```diff
@@ -12,7 +12,7 @@ set(DEFAULT_BC_COMPILER_FLAGS
   -Wno-gnu-anonymous-struct -Wno-return-type-c-linkage
   -Wno-gnu-zero-variadic-macro-arguments -Wno-nested-anon-types
   -Wno-extended-offsetof -Wno-gnu-statement-expression -Wno-c99-extensions
-  -Wno-ignored-attributes -mtune=generic -fno-vectorize -fno-slp-vectorize
+  -Wno-ignored-attributes -fno-vectorize -fno-slp-vectorize
```

Comment:
The new `clang-15` didn't seem to like this flag when compiling the runtime for SPARC. I don't think it's really necessary so I've removed it.